website/_redirects

```diff
@@ -0,0 +1,4 @@
+# Redirects from what the browser requests to what we serve
+/docs/*     	/v0.5.5/:splat
+/docs/provisioner-crd/	        /v0.5.5/provisioner/  		
```

Comment:
Doesn't line 2 cover this case?

Reply:
Is this netlify-specific?

Reply:
a while ago we renamed `provisioner-crd` to `provisioner` so this needs a separate redirect

Reply:
yes, this is netlify specific

Reply:
Can we make the destination here `/docs/provisioner` instead of the version specific path?